docker-compose.yml

```diff
@@ -2,18 +2,32 @@ version: '2'
 
 services:
   db:
+    restart: 'always'
     image: 'postgres:9.5'
     ports:
       - '5432:5432'
     volumes:
       - 'db:/var/lib/postgresql/data'
+
   app:
+    restart: 'always'
     depends_on:
       - 'db'
     build: .
     ports:
       - '3000:3000'
-    env_file: '.env'
+    volumes:
+      - '.:/test_track'
+    volumes_from:
+      - bundler_cache
+    command: ./script/serve
+    environment:
+      - DATABASE_URL=postgresql://postgres@db/test_track_development
```

Comment:
@samandmoore can give his perspective, but basically the thinking is that if we're looking at this for local development then the user will have the repo cloned anyway, so it's not a problem to change this value. Also, this keeps all of the docker configuration in one place instead of spreading it to a `.env` file, which would potentially be used by a Rails project using dotenv.